### PR TITLE
Developer Tools week 3 days 4 update

### PR DESCRIPTION
**Pull request by captainbrosset**

### CODE REVIEW

File: docs/30DaysOfPWA/dev-tools/03.md [hunk 1]
```diff
@@ -16,3 +16,137 @@ Welcome to **Week 3 Day 3** of #30DaysOfPWA! Want to learn more about this proje
...
+It's not always possible though and DevTools comes with a very useful **Device Emulation** tool you can use to check what your app would look and feel like on other devices.
+
+To start **Device Emulation**:
+
+* Open DevTools by pressing `F12` or right-clicking the page and selecting `Inspect`.
+* Click the **Toggle device emulation** button in the top-left corner of DevTools.
+
+![Screenshot of a tab in Microsoft Edge with DevTools open and the device emulation tool started, making the page appear as if rendered on a mobile device](_media/device-mode.png)
+
+Choose the devices you want to emulate in the **Dimensions** drop-down menu. When you do, the viewport resizes to match the emulated device's screen, and your cursor acts like a touch input if the emulated device supports it. If you want, you can also click **Edit** in the menu and create your own devices.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/03.md:45]: You won't get device emulation option in Edge if the PWA is running separately from the browser (like with the PWA Starter), we should probably add a note of this. I'm not sure of the exact display modes or situations where you gain and lose device emulation.

**Review by zateutsch — COMMENTED** [review]
These both look pretty good to me. Just one clarity thing in the Emulation section.

Thanks, Patrick!

**Comment by beth-panx** [comment]
Hey @captainbrosset - Can you break this into 2 PRs? So that we can merge on two different days.

**Comment by captainbrosset** [comment]
Will do. Chait and I have been working on another repo before, and I wanted to make sure I brought all of our commits over (with the right authors). The way I did this led to 1 PR. But I will split it in 2 now.

File: docs/30DaysOfPWA/dev-tools/03.md (hunk 1 above)
Comment (captainbrosset) [reply docs/30DaysOfPWA/dev-tools/03.md:45]: Great point! I will add this.

**Comment by captainbrosset** [comment]
@beth-panx done. This PR is now only about week 3 - day 4.

File: docs/30DaysOfPWA/dev-tools/03.md (hunk 1 above)
Comment (captainbrosset) [reply docs/30DaysOfPWA/dev-tools/03.md]: This will be fixed in #64 as day 3 moved there.

**Comment by captainbrosset** [comment]
Actually let me close this one. I have 2 new PRs, with the right branch names, so it's less confusing.

* week 3 day 3: #64 
* week 3 day 4: #65

**Pull request closed without merging**